assets/sass/local.scss

```diff
@@ -251,3 +251,35 @@
 .bapv-force-overflow {
  overflow-wrap: anywhere;
 }
+
+.bapv-timetable-dates {
+  display: flex;
+  justify-content: space-between;
+  max-width: 600px;
+}
+
+.bapv-timetable-dates__date {
+  flex-basis: 10%;
+  padding: govuk-spacing(2) govuk-spacing(2);
```

Comment:
Can just be `padding: govuk-spacing(2);`

Reply:
Fixed!